app/services/users.js

```diff
@@ -37,3 +37,26 @@ exports.getUsers = async (limit, offset) => {
     throw databaseError(error.message);
   }
 };
+
+exports.updateUser = async (userId, userData) => {
+  try {
+    const response = await User.update(userData, { where: { id: userId } });
+    return response;
+  } catch (error) {
+    logger.error(error.name);
+    throw databaseError(error.message);
```

Comment:
Do not throw database errors. It could contain sensitive information and it usually reaches directly to the client

Reply:
ok, I changed whit a custom message